boards/arm/nrf9160_innblue21/doc/index.rst

```diff
@@ -0,0 +1,105 @@
+.. _nrf9160_innblue21:
+
+nRF9160-INNBLUE21
+#################
+
+Overview
+********
+
+The "innblue v2.1" is a cellular IoT sensor development board, which
+is based on nRF9160 SiP featuring NB-IoT and LTE-M connectivity.
+
+.. figure:: img/nrf9160_innblue21.png
```

Comment:
This looks like a very nice board. I may buy one myself :).

Reply:
thanks! indeed, nice :) @mbolivar-nordic if you are in Trondheim few boards should be there :)

Reply:
I am not. Are they commercially available? I can't tell from the docs or innblue website.